rest-api/dao/participant_summary_dao.py

```diff
@@ -163,11 +166,19 @@ def _add_order_by(self, query, order_by, field_names, fields):
 
   def make_query_filter(self, field_name, value):
     """Handle HPO and code values when parsing filter values."""
-    if field_name == 'hpoId':
+    if field_name == 'hpoId' or field_name == 'awardee':
       hpo = self.hpo_dao.get_by_name(value)
       if not hpo:
         raise BadRequest('No HPO found with name %s' % value)
+      if field_name == 'awardee':
+        field_name = 'hpoId'
       return super(ParticipantSummaryDao, self).make_query_filter(field_name, hpo.hpoId)
+    if field_name == 'organization':
```

Comment:
What about site?

Reply:
Site is covered by the _SITE_FIELDS list